File: programming-concepts/selection/python/test_score_if_else.py
# Isaac Computer Science
# Usage licensed under the Open Government Licence v3.0

def main():
    print("Enter a test score")
    user_input = input()
    test_score = int(user_input)

    if test_score >= 60:
        print("Pass")
    else:
        print("Retry")

    print("End of program")


# This code will run if this file is executed directly
# (i.e. not called by another program)
if __name__ == '__main__':
    main()

File: programming-concepts/selection/c-sharp/test_score_if_else.cs
/*
Raspberry Pi Foundation
Developed to be used alongside Isaac Computer Science, part of the National Centre for Computing Education
Usage licensed under CC BY-SA 4

Note: This file is designed to be copied out and compiled on your machine.
In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file.
To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Visual Studio, for example)
3. Change the namespace to match your project (if necessary)
4. Compile the program
5. Run the program
*/

using System;

namespace IsaacCodeSamples
{

    class Selection
    {
        // The Main method is the entry point for all C# programs
        public static void Main() {
            Console.WriteLine("Enter a test score");
            string userInput = Console.ReadLine();
            int testScore = Int32.Parse(userInput);

            if (testScore >= 60) {
                Console.WriteLine("Pass");
            }
            else {
                Console.WriteLine("Retry");
            }

            Console.WriteLine("End of program");
        }
        

    }
}
